lib/iron_bank/describe/excluded_fields.rb

```diff
@@ -87,9 +88,65 @@ def valid_query?
         info "Successful query for #{object_name}"
 
         true
-      rescue IronBank::InternalServerError, IronBank::BadRequestError => e
+      rescue IronBank::BadRequestError => e
         @last_failed_fields = extract_fields_from_exception(e)
```

Comment:
let's do some tidying first. extract `#extract_fields_from_exception(e)` into a service class, so this will now read as:
```rb
@last_failed_fields = IdentifyFromMessage.call(e.message)
```
and the service class will have the form:
```rb
module IronBank
  module Describe
    class ExcludedFields::ExtractFromErrorMessage
      def self.call(exception)
        new(exception).call
      end
    
      def call
        # ... implementation here ...
      end

      private

      attr_reader :message

      def initialize(message)
        @message = message
      end
    end
  end
end